ee/desktop/menu/action_open_url.go

```diff
@@ -12,12 +12,21 @@ type actionOpenURL struct {
 	URL string `json:"url"`
 }
 
-func (a actionOpenURL) Perform(m *menu) {
-	if err := open(a.URL); err != nil {
+func (a actionOpenURL) Perform(m *menu, parser textParser) {
```

Comment:
Why do we have to pass in the parser if we already have `menu.parser`?

Reply:
Good catch! This was something left over from my first iteration, removed now!